README.md

```diff
@@ -1 +1,21 @@
-interactive-make
+# Interactive Make for Laravel 5.4
+
+## Getting Started
+
+To get started you must first install the package from composer.
+
+```
```

Comment:
append `bash` here and you'll get syntax highlighting (if available) and it'll be more a11y 🙂